"TypeError: Cannot read property 'unknownOptions' of undefined" and "Error: Cannot find module '../cli'" saga

Just do `npm i @elastic/plugin-helpers@7` and then `npm start`

```
./node_modules/.bin/plugin-helpers start
Task "start" failed:

TypeError: Cannot read property 'unknownOptions' of undefined
    at program.command.description.on.action.createCommanderAction (/opt/dev/bitbucket.org/qaap-uniqcast/statia/kibana/plugins/uauth/node_modules/@elastic/plugin-helpers/cli.js:17:22)
    at Promise.resolve.then (/opt/dev/bitbucket.org/qaap-uniqcast/statia/kibana/plugins/uauth/node_modules/@elastic/plugin-helpers/lib/commander_action.js:7:33)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
```

I have kibana source in a folder parallel to the plugin one where this is installed. It is a symlink folder.

If a plugin folder is not a symlink, then I get
```
uauth@0.1.0 start /opt/dev/github.com/mkozjak/uauth                                                                                                                             
> plugin-helpers start                                                                                                                                                            
                                                                                                                                                                                  
module.js:549                                                                                                                                                                     
    throw err;
    ^

Error: Cannot find module '../cli'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/opt/dev/github.com/mkozjak/uauth/node_modules/.bin/plugin-helpers:9:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

"@elastic/plugin-helpers": "^7.1.7"

--- a/lib/commander_action.js
+++ b/lib/commander_action.js
@@ -1,10 +1,13 @@
 const run = require('./run');
 
 module.exports = function createCommanderAction(taskName, getOptions = () => {}) {
-  return (command, ...args) => {
+  return (...args) => {
+    // command is the last arg passed by commander, but we move it to the front of the list
+    const command = args.pop();
+
     return Promise
       .resolve()
-      .then(() => run(taskName, getOptions(...args)))
+      .then(() => run(taskName, getOptions(command, ...args)))
       .catch((error) => {
         process.stderr.write(`Task "${taskName}" failed:\n\n${error.stack || error.message}\n`);
         process.exit(1);

